helm-charts/splunk-otel-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: splunk-otel-collector
-version: 0.24.12
+version: 0.24.13
```

Comment:
There are 5 PRs open at this time.  What is the reason to include a version bump along with the change itself as opposed to a separate release commit?

Reply:
In the helm chart, we don't follow the rule to have a release commit so far. I don't see a reason to do that at the moment. Just to introduce a change that can be utilized right away. Git tags can be used to track history of releases instead of release commits.

@jrcamp what's your thoughts?

Reply:
But I'm open to having release commits if everyone agrees

Reply:
I think bumping the version in the same PR is fine. If there are multiple changes ready to go out it would be nice to group them into the same release just to avoid superfluous releases. I doubt it's bothering anybody though.